wfc3tools/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - scipy
     - python x.x
 source:
-    git_tag: 1.3.1
+    git_tag: 1.3.3
```

Comment:
okay, I'll look tomorrow, and just fix it to be uniform. I remember the policy, but I forget when I'm working on astropy and stsci packages simulaneously, the v snuck in there, it was in the draft version notes and github tagged it when I published. I'll go check the draft hstcal notes to make sure it's off there too.